js/vulnerability.js

```diff
@@ -116,14 +110,16 @@ function sortVulnerabilities(listElement) {
 function getStorageVulnsAndRender(items) {
   chrome.runtime.sendMessage("getStoredTraces", (response) => {
     if (!!response && !!response.traces && response.traces.length > 0) {
-      $("#no-vulnerabilities-found-on-page").hide();
-      $("#vulnerabilities-found-on-page").show();
+      setDisplayNone(document.getElementById("no-vulnerabilities-found"));
```

Comment:
I'm fine with chucking jQuery, but there's no way this is more clear than what it is replacing. 🚜

Reply:
Added some methods to the HTMLElement prototype

Reply:
Is that an acceptable JS practice? If this were Ruby, for instance, I'd probably protest.

Reply:
Yeah probably not the best idea if we ever use a library in the future - how about something like:
```
function setElementDisplay(element, displayValue)
```